fuel/src/main/kotlin/com/github/kittinunf/fuel/core/Encoding.kt

```diff
@@ -74,9 +75,13 @@ class Encoding : Fuel.RequestConvertible {
     private fun queryFromParameters(params: List<Pair<String, Any?>>?): String {
         return params?.let {
             params.filterNot { it.second == null }
-                    .mapTo(arrayListOf<String>()) { "${it.first}=${it.second}" }
+                    .mapTo(ArrayList()) { "${it.first}=${it.second}" }
```

Comment:
We should stick with Kotlin types. Using `mutableListOf()` should be better option.